### DISCUSSION TITLE
Crash during Sephiroth Black Materia video in Northern Crater

### DISCUSSION THREAD

**Discussion by ChiefDank** [julianxhokaxhiu/FFNx#458] [Q&A] [answered]
I am getting stuck here, just after the whirlwind maze. Please let me know what I need to attach for you to take a look.

**Comment by julianxhokaxhiu** [comment, marked as answer]
Most probably your crash is related to 7th, but here no support is provided. Feel free to approach the relative Discord channels.

Also consider updating to the latest canary, before reporting. chances are this has been fixed.
